src/lua/LuaGame.cpp

```diff
@@ -738,6 +739,75 @@ static int l_game_get_parts_from_date_time(lua_State *l)
 	return 6;
 }
 
+template<typename T>
+static int l_game_get_config(lua_State *l)
+{
+	// Restrict the template function to only a few types
+	static_assert(
+		std::is_same<T, bool>::value ||
+		std::is_same<T, int>::value ||
+		std::is_same<T, float>::value ||
+		std::is_same<T, std::string>::value
```

Comment:
For consistency, these should use `std::is_same_v<T, type>`.
```suggestion
		std::is_same_v<T, bool> ||
		std::is_same_v<T, int> ||
		std::is_same_v<T, float> ||
		std::is_same_v<T, std::string>
```